tests/foreman/ui/test_computeresource_gce.py

```diff
@@ -81,6 +91,7 @@ def test_positive_default_end_to_end_with_custom_profile(
     cr_description = gen_string('alpha')
     new_org = entities.Organization().create()
     new_loc = entities.Location().create()
+    download_cert
```

Comment:
Are you trying to create above as var ?

Reply:
No ... its the fixture I am calling : https://github.com/SatelliteQE/robottelo/pull/7041/files#diff-90cf96329c7bb8b7528a8678e8224b06R61

Reply:
as discussed , it already calling here:  https://github.com/SatelliteQE/robottelo/blob/317ea21e1cbe94a00c32edcb236d993a75988e03/tests/foreman/ui/test_computeresource_gce.py#L70
please remove from here .

Reply:
addressed in latest push